src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -115,6 +115,22 @@ def get_action(self, observation):
                     dict(mean=dist.mean.squeeze(0).numpy(),
                          log_std=(dist.variance**.5).log().squeeze(0).numpy()))
 
+    def forward(self, *obervations):
+        """Forward method.
+
+        Args:
+            obervations (torch.Tensor): Observation from the environment.
+                Shape is :math:`env_spec.observation_space`.
```

Comment:
`env_spec.observation_space` is not a shape. this also includes no information on the batch dimension (if any). please see `g/_dtypes.py` for examples of how to document shapes.

Reply:
fixed.